web/web-base/src/main/java/org/visallo/web/routes/vertex/VertexSetProperty.java

```
@@ -193,11 +193,11 @@ public ClientApiVertex handle(
                         user,
                         authorizations);
             } else {
-                if (valueStr == null) {
+                if (valueStr == null && valuesStr == null) {
```

Should this be `||`?

---

we should throw an error if they are both blank. either `valueStr` or `valuesStr` should be filled in